From 2ff1e88665fdbdede637e4f8aff21e772b8a1ced Mon Sep 17 00:00:00 2001
From: Johnny Mnemonic <jm@machine-hall.org>
Date: Fri, 3 Jan 2025 23:59:48 +0100
Subject: [PATCH 2/2] Introduce own processor type for Ski

This replaces the up until now used McKinley and is a SDM 2.2 capable
Itanium processor as per [1]. Processor features:

```
features   : branchlong, 16-byte atomic ops
```

...are identical to Itanium 2 9000 (Montecito) processors - hence the
naming - and exceed the features of for example a Madison processor:

```
features   : branchlong
```

[1]: https://github.com/trofi/ski/blob/master/NEWS#L63
---
 arch/ia64/kernel/setup.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/ia64/kernel/setup.c b/arch/ia64/kernel/setup.c
index 5f44b759caf7..54fce18d4770 100644
--- a/arch/ia64/kernel/setup.c
+++ b/arch/ia64/kernel/setup.c
@@ -793,6 +793,7 @@ get_model_name(__u8 family, __u8 model)
 			case 0: memcpy(brand, "McKinley", 9); break;
 			case 1: memcpy(brand, "Madison", 8); break;
 			case 2: memcpy(brand, "Madison up to 9M cache", 23); break;
+			case 3: memcpy(brand, "MonteSkito", 11); break;
 		}
 	}
 	for (i = 0; i < MAX_BRANDS; i++)
-- 
2.47.2

